downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -0,0 +1,27 @@
+//this module appears in assembly-collections-and-content-signing-in-pah
+
+[id="proc-downloading-signature-public-keys"]
+
+= Downloading signature public keys
+
+After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
+
+.Procedure
+
+. Log in to your private {HubName} instance in the {HubName} UI.
+. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+
+. Select the menu icon, click btn:[Download Key] to download the public key.
```

Comment:
These seem more like two steps to me (suggestion only):

1. Select the three dots or Kebab menu icon to the right of your public key.
2. Click btn:[Download Key] to download the public key.

Or you could combine it into one sentence: Select the Kebab menu and click btn:[Download Key] to download the public key.

However, I have another suggestion to combine things by having the customer select one of the 3 ways they can download a public key. See below.

Reply:
Thanks, @jself-sudoku for the suggestions.